services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -174,6 +177,97 @@ class FlowHelper {
         } || existingFlows*.id.contains(newFlow.id)
     }
 
+    /**
+     * Check that all needed rules are created for a flow with protected path
+     *
+     * @param flowId
+     */
+    void verifyRulesOnProtectedFlow(String flowId) {
```

Comment:
Method name or description should clearly state that rules are being verified for both main path and protected path. Currently it looks like we just verify the protected path, `needed rules are created` is pretty poor description to answer my question

Reply:
updated